onnxscript/onnx_types.py

```diff
@@ -203,3 +239,22 @@ def onnx_type_to_onnxscript_repr(onnx_type: onnx.TypeProto) -> str:
 
 # Currently, only tensor types are supported. Need to expand support for other ONNX types.
 ONNXType = TensorType
+
+ALL_TENSOR_TYPES = (
```

Comment:
Could you please sort them in alphabetical order for better readness?

Reply:
Done